meteor/client/ui/FloatingInspectors/MicFloatingInspector.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import Moment from 'react-moment'
+
+import { FloatingInspector } from '../FloatingInspector'
+import { ScriptContent } from '@sofie-automation/blueprints-integration'
+
+interface IProps {
+	typeClass?: string
+	showMiniInspector: boolean
+	itemElement: HTMLDivElement | null
+	floatingInspectorStyle: React.CSSProperties
+	content: ScriptContent
+}
+
+const BREAK_SCRIPT_BREAKPOINT = 620
+const SCRIPT_PART_LENGTH = 250
+
+export function MicFloatingInspector(props: IProps) {
+	const { t } = useTranslation()
+
+	let startOfScript = (props.content && props.content.fullScript) || ''
+	let cutLength = startOfScript.length
+	if (startOfScript.length > SCRIPT_PART_LENGTH) {
+		startOfScript = startOfScript.substring(0, startOfScript.substr(0, SCRIPT_PART_LENGTH).lastIndexOf(' '))
+		cutLength = startOfScript.length
```

Comment:
This is the same assignment as the initial assigment, isn't it?

Reply:
Well, no, because at that stage, the `startOfScript.length` will be different, because `startOfScript` has been reassigned.

Reply:
Yes, of course :)